tests/integration/models/reddit/test_subreddit.py

```diff
@@ -1195,3 +1195,17 @@ def test_revisions(self, _):
                 assert isinstance(revision['author'], Redditor)
                 assert isinstance(revision['page'], WikiPage)
             assert count == 4
+
+
+class TestSubredditEmoji(IntegrationTest):
+    @mock.patch('time.sleep', return_value=None)
+    def test__iter(self, _):
+        self.reddit.read_only = False
+        subreddit = self.reddit.subreddit(
+            pytest.placeholders.test_subreddit)
+        with self.recorder.use_cassette('TestSubredditEmoji.iter'):
```

Comment:
Can you rename this cassette to `'TestSubredditEmoji.test__iter` so that it matched the class and method name exactly? You'll also have to rename the cassette file itself.